server/cli/launch.py

```diff
@@ -16,7 +16,7 @@
 @click.option(
     "--layout",
     "-l",
-    type=click.Choice(["umap", "tsne", "draw_graph_fa", "draw_graph_fr"]),
+    type=click.Choice(["umap", "tsne", "draw_graph_fa", "draw_graph_fr", "diffmap", "phate"]),
```

Comment:
Why do we restrict the user to this set of layouts?   I would advocate for letting the user create and view their own custom layout, if they are so inclined.  Definitely an advanced user feature, but super easy to add and very useful.

Reply:
Hmm... I've thought of the same thing before. I like how having options leads to discoverability and protects against non-viewable layouts. But I also agree that advanced users shouldn't be artificially limited.

Reply:
All good points. As of now, I believe we support / enable all embeddings implemented in scanpy. Any additional / "custom" layouts would have to be computed externally and then attached manually to the anndata structure. Could happen, but I suspect even for advanced users it's not super common? Definitely something we should think about if / when we're ready to support `launch` input from other ecosystems, though.